build.gradle

```diff
@@ -97,12 +102,14 @@ subprojects { subproject ->
   }
 
   if (RELEASE_MODULES.contains(subproject.name)) {
-    subproject.apply from: "${rootDir}/gradle/mvn-push.gradle"
+    subproject.apply plugin: 'com.jfrog.bintray'
```

Comment:
Already applied in the `gradle-bintray.gradle`.

Reply:
Removed